models/enterprise_sonic/snmp/merged_example_01.txt

```diff
@@ -1,31 +1,29 @@
 # Using merged
-
-# Before state:
-# -------------
 #
-#rno-dctor-1ar01c01sw02# show snmp-server community
+# Before state:
+#---------------
 #
-#         Community Name                     Group Name
-#-------------------------------- --------------------------------
-#private                          ro
-#public                           ro
+# show running-configuration | grep snmp
+# (no snmp-server configuration present)
 #
 
-- name: Merge a single SNMP community
-  sonic_snmp:
-    config:
-      - community: "public123"
-        access: "ro"
-    state: merged
+# - name: Merge snmp-server configuration
+#   sonic_snmp:
+#       config:
+#           users:
+#               - name: user1
+#                 auth: 
```

Comment:
Update the yaml hierarchy here to match the current model ".yaml" file (and argspec). Use "auth_type" and "key".